### PR TITLE
engine: add release-notes for 20.10.19

### PR DESCRIPTION
**Pull request by thaJeztah**
https://github.com/moby/moby/releases/tag/v20.10.19

### CODE REVIEW

**Comment by thaJeztah** [comment]
@usha-mandya @tianon ptal

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -15,6 +15,31 @@ for Docker Engine.
...
+of Docker Compose.
+
+### Builder
+
+- Fix an issue that could result in a panic during `docker builder prune` or
+  `docker system prune` [moby/moby#44122](https://github.com/moby/moby/pull/44122){:target="_blank" rel="noopener"}.
+
+### Daemon
+
+- Fix an bug where using `docker volume prune` would remove volumes that were
```
Comment (usha-mandya) [inline engine/release-notes/index.md:31]:
```suggestion
- Fix a bug where using `docker volume prune` would remove volumes that were
```

Comment (thaJeztah) [reply engine/release-notes/index.md]: Ah, thanks! I guess I had `Fix an issue` before, and changed it to bug 😅 

Either way; updated, PTAL 🤗

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**